web/src/popover_menus.js

```diff
@@ -705,4 +799,129 @@ export function initialize() {
             popover_instances.all_messages = undefined;
         },
     });
+
+    $("body").on("click", "#schedule-confirm", (e) => {
+        schedule_message_to_custom_date();
+
+        e.preventDefault();
+        e.stopPropagation();
+    });
+
+    function show_schedule_confirm_button(send_at_time, not_from_flatpickr) {
+        if (!send_at_time) {
+            return;
+        }
+        if (!not_from_flatpickr) {
+            selected_send_later_id = "send-later-custom-input";
+            send_at_time = format(new Date(send_at_time), "MMM d yyyy h:mm a");
+        }
```

Comment:
Can you add comments documenting what these `not_from_flatpickr` and `sent_at_time` parameters mean? I'm not sure why the former would ever be null, or why `not_from_flatpickr` is a good parameter name.

Reply:
Okay, added comments to explain them.